cargo-dist/src/tasks.rs

```diff
@@ -2484,6 +2488,115 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
         Ok(())
     }
 
+    fn add_pkg_installer(&mut self, to_release: ReleaseIdx) -> DistResult<()> {
+        if !self.local_artifacts_enabled() {
+            return Ok(());
+        }
+
+        // Clone info we need from the release to avoid borrowing across the loop
+        let release = self.release(to_release).clone();
+        let variants = release.variants.clone();
+        let bin_aliases = release.bin_aliases.clone();
+        let checksum = release.checksum;
+
+        // Make a pkg for every darwin platform
+        for variant_idx in variants {
+            let variant = self.variant(variant_idx);
+            let binaries = variant.binaries.clone();
+            let target = &variant.target;
+            if !target.contains("darwin") {
+                continue;
```

Comment:
Normally we warn here, I think?

Reply:
I was following the pattern from the MSI, which doesn't warn here.